plugin.json

```diff
@@ -1,9 +1,9 @@
 {
     "id": "object-detection-gpu",
-    "version": "0.1.6",
+    "version": "0.1.7",
     "meta": {
         "label": "Object detection in images (GPU)",
```

Comment:
should we add `partially superseded` in the plugin title ? I don't have a strong opinion on that